tests/regressions/lcos/CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ set(async_callback_non_deduced_context_PARAMETERS THREADS_PER_LOCALITY 4)
 set(broadcast_unwrap_future_2885_PARAMETERS LOCALITIES 2 THREADS_PER_LOCALITY 4)
 set(broadcast_wait_for_2822_PARAMETERS LOCALITIES 2 THREADS_PER_LOCALITY 4)
 set(channel_2916_FLAGS DEPENDENCIES iostreams_component)
-set(channel_2916_PARAMETERS LOCALITIES 2 THREADS_PER_LOCALITY 4)
+set(channel_2916_PARAMETERS THREADS_PER_LOCALITY 4)
```

Comment:
why did you remove distributed testing here?

Reply:
Because the test itself does not do any distributed work.

Reply:
Ah, yes sure...